types/swift/$screen.swift

```diff
@@ -0,0 +1,125 @@
+//
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+// GENERATED FILE, DO NOT EDIT. FOR MORE INFORMATION VISIT
+// https://github.com/matrix-org/matrix-analytics-events/
+
+/// Triggered when the user changed screen
+extension AnalyticsEvent {
+    public struct $screen: AnalyticsEventProtocol {
```

Comment:
@t3chguy This wouldn't be a valid class name in Swift. Could we leave the filename as `Screen.json` and only update the `eventName` property?

Reply:
This wouldn't match the guidelines set out by this repo

https://github.com/matrix-org/matrix-analytics-events#guidelines-for-creating-new-event-schema

> The filename should match the event's name, including case.

But yes it is an exception we will have to make.

Reply:
Ah that's annoying to have guidelines and then go an break them 😕
Maybe we should update that sentence to mention dropping special characters, and the if necessary on the mobile side we could update the generator to always upper case the first letter of the class name.